share/dictionary/dhcpv4/dictionary.rfc2563

```diff
@@ -0,0 +1,15 @@
+# -*- text -*-
+# Copyright (C) 2022 The FreeRADIUS Server project and contributors
+# This work is licensed under CC-BY version 4.0 https://creativecommons.org/licenses/by/4.0
+# Version $Id$
+##############################################################################
+#
+#  RFC2563 - DHCP Option to Disable Stateless Auto-Configuration in IPv4
+#            Clients
+#
+#  $Id$
+#
+##############################################################################
+
+# DHCP Auto-Configuration
+ATTRIBUTE	Auto-Config				116	uint8
```

Comment:
This should probably be `bool`, as the RFC defines it to be 0/1.

Reply:
Corrected.